### PR TITLE
Improve aws preset kube-up/kube-down

### PR DESCRIPTION
**Pull request by Hong-Chang**
1.	Move preset machines to be the same vpc of Jenkins slave4, then using internal ip.
2.	Move machine clean-up code to a newly created repo
3.     Don't wait while loop forever
4.     Add error tolerance for clean up

**What type of PR is this?**
/kind feature

### CODE REVIEW

File: cluster/aws/configure-vm.sh [hunk 1]
```diff
@@ -778,9 +778,16 @@ function setup-kubernetes-worker() {
...
 
   pushd /etc/kubernetes
+  local attempt=0
   until wget http://$KUBE_MASTER_IP:8085/join_string &>/dev/null; do
     echo "Waiting for kubeadm join command .."
     sleep 5
+    attempt=$(($attempt+1))
+    if (( attempt > 120 )); then
+      echo
+      echo "Waiting for kubeadm join command failed after 10 minutes elapsed"
```
Comment (sonyafenge) [inline cluster/aws/configure-vm.sh:788]: 10 minutes are too long, it should be done within 5 minutes.

File: cluster/aws/util.sh [hunk 2]
```diff
@@ -805,6 +806,12 @@ function wait-for-instance-state {
       echo "Waiting for instance ${instance_id} to be ${state} (currently ${instance_state})"
       echo "Sleeping for 3 seconds..."
       sleep 3
+      attempt=$(($attempt+1))
+      if (( attempt > 200 )); then
+        echo
+        echo "Waiting for instance state failed after 10 minutes elapsed"
```
Comment (sonyafenge) [inline cluster/aws/util.sh:812]: 10 minutes are too long, it should be done within 5 minutes.

File: cluster/aws/util.sh [hunk 3]
```diff
@@ -1498,6 +1503,12 @@ function wait-master() {
     fi
     printf "."
     sleep 5
+    attempt=$(($attempt+1))
+    if (( attempt > 240 )); then
+      echo
+      echo "Waiting for master failed after 20 minutes. There might be some uncaught error during start."
```
Comment (sonyafenge) [inline cluster/aws/util.sh:1509]: 20 minutes are too long, it should be done within 5 minutes.

Comment (Hong-Chang) [reply cluster/aws/util.sh]: Updated.

File: cluster/aws/util.sh (hunk 2 above)
Comment (Hong-Chang) [reply cluster/aws/util.sh]: Updated.

File: cluster/aws/configure-vm.sh (hunk 1 above)
Comment (Hong-Chang) [reply cluster/aws/configure-vm.sh]: Updated.

File: cluster/aws/configure-vm.sh [hunk 4]
```diff
@@ -778,9 +778,16 @@ function setup-kubernetes-worker() {
   echo "Setting up kubernetes worker: version $KUBE_VER master IP: $KUBE_MASTER_IP."
 
   pushd /etc/kubernetes
+  local attempt=0
   until wget http://$KUBE_MASTER_IP:8085/join_string &>/dev/null; do
     echo "Waiting for kubeadm join command .."
     sleep 5
+    attempt=$(($attempt+1))
+    if (( attempt > 120 )); then
```
Comment (jshaofuturewei) [inline cluster/aws/configure-vm.sh]: Trying 120 times.

We prefer elapse time to attempts since it might stuck somewhere and does not proceed.

Comment (Hong-Chang) [reply cluster/aws/configure-vm.sh]: Updated.

**Pull request merged**